qutebrowser/browser/webengine/webview.py

```diff
@@ -173,7 +173,12 @@ def _set_bg_color(self):
         col = config.val.colors.webpage.bg
         if col is None:
             col = self._theme_color
-        self.setBackgroundColor(col)
+        if self.view() is not None:
+            if col.alpha() < 255:
+                self.view().setAttribute(Qt.WA_TranslucentBackground)
+            else:
+                self.view().resetAttribute(Qt.WA_TranslucentBackground)
```

Comment:
>resetAttribute

I don't think this is a real thing. It certainly raised an AttributeError for me. It [looks](http://doc.qt.io/qt-5/qwidget.html#setAttribute) like you can use `setAttribute(..., False)`

Reply:
I modified it, thanks for your help :)